pilot/pkg/model/authorization.go

```diff
@@ -50,13 +50,22 @@ func GetAuthorizationPolicies(env *Environment) *AuthorizationPolicies {
 
 	policies := env.List(gvk.AuthorizationPolicy, NamespaceAll)
 	sortConfigByCreationTime(policies)
+
+	stats := make(map[string]int)
```

Comment:
nit: `stats` is a somewhat odd name. Maybe `policyCount` or something?

Reply:
thanks, done.